packages/core/src/queries/connector.test.ts

```diff
@@ -27,13 +27,16 @@ describe('connector queries', () => {
 
   it('findAllConnectors', async () => {
     const rowData = { id: 'foo' };
-    const expectSql = sql`
-      select ${sql.join(Object.values(fields), sql`, `)}
-      from ${table}
+
+    /* eslint-disable sql/no-unsafe-query */
+    const expectSql = `
+      select "id", "enabled", "config", "created_at"
+      from "connectors"
+      order by "enabled" desc, "id" asc
```

Comment:
not sure whether we can specify the order in the original SQL command instead of rewriting it?

Reply:
better like this

```ts
export const findAllConnectors = async (orderBy?: OrderBy<Connector>) =>
  findConnectorMany({ orderBy });
```